include/aikido/planner/dart/ConfigurationToTSR.hpp

```diff
@@ -24,7 +24,11 @@ class ConfigurationToTSR : public Problem
   /// current state of when called.
   /// \param[in] endEffectorBodyNode BodyNode to be planned to move to a desired
   /// TSR.
-  /// \param[in] maxSamples Maximum number of TSR samples to plan to.
+  /// \param[in] maxSamplingTries Maximum number of times to try when
```

Comment:
Maybe specify that this is the parameter to the underlying sample generator, similarly to:
https://github.com/personalrobotics/aikido/blob/92befdfe0af887aa23bf817b372c87b3933b789c/include/aikido/constraint/dart/InverseKinematicsSampleable.hpp#L32